test/services/tagged_organisations_test.rb

```diff
@@ -0,0 +1,69 @@
+require 'test_helper'
+require './test/support/custom_assertions.rb'
+
+describe TaggedOrganisations do
+  describe '#fetch' do
+    it 'return tagged organisations' do
+      search_results = {
```

Comment:
Seems like this could be refactored into either a helper method or `let` vars.

Reply:
Done https://github.com/alphagov/collections/pull/586/files#diff-62314a7230a7329a26b5cb721a328944